internals/garbage-collector.rst

```diff
@@ -1,14 +1,17 @@
+.. _garbage-collector:
 .. _gc:
+.. _garbage_collector:
```

Comment:
Are these for backward-compatibility with existing links?

Reply:
Yes